README.md

```diff
@@ -131,4 +132,4 @@ Please see our [guidelines](CONTRIBUTING.md) for contributing to the driver.
 * Testo                     test1@doramail.com   
 * Zhmayev Yaroslav          https://github.com/salaros
 
-If you have contributed and we have neglected to add you to this list please contact one of the maintainers to be added to the list (with apologies).
\ No newline at end of file
+If you have contributed and we have neglected to add you to this list please contact one of the maintainers to be added to the list (with apologies).
```

Comment:
Can't see the change on this line.

Reply:
Thanks for the catch! Newline shenanigans. Perhaps worth standardizing on a "new line at end of file" convention one of these days.